### PR TITLE
Feat/delete archive

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. feature : delete note and move to trash
2. feature: navigate to trash and see lists of deleted notes
3. feature: archive note
4. feature: navigate to archive and see list of archived notes
5. feature: edit and delete archive notes

### CODE REVIEW

**Comment by harshitpaliwal95** [comment]
hey Shikha
- when i create new note nothing is happening on my side?

https://user-images.githubusercontent.com/60725346/161481569-ae83f13b-3a89-49d1-87c8-e4bf45f26c38.mov

**Comment by SHIKHASINGH1506** [comment]
> hey Shikha
> 
>     * when i create new note nothing is happening on my side?
> 
> 
> Screen.Recording.2022-04-04.at.11.18.01.AM.mov

I guess there's some deploy issue. Let me check on that.

**Review by ark2002 — APPROVED** [review]
-The folder structure looks great
-The UI looks clean 
- functionality working great aswell .

File: src/backend/controllers/ArchiveController.js [hunk 1]
```diff
@@ -66,7 +66,41 @@ export const restoreFromArchivesHandler = function (schema, request) {
...
+  } catch (error) {
+    return new Response(
+      500,
+      {},
+      {
+        error,
+      }
+    );
+  }
+}
```
Comment (devdeadviz) [inline src/backend/controllers/ArchiveController.js:106]: Nice implementtaation.

**Review by devdeadviz — APPROVED** [review]
The code is clean and you have followed all the best practices. Good work!

**Pull request merged**